crates/networking/p2p/sync.rs

```diff
@@ -285,14 +285,12 @@ impl Syncer {
             current_head = last_block_hash;
             current_head_number = last_block_number;
 
-            // If the sync head is less than 64 blocks away from our current head switch to full-sync
+            // If the sync head is not 0 we search to fullsync
```

Comment:
IMO, if the sync head is 0 pero is less than X blocks away, we should also not do snap sync. Snap sync is for 1M blocks distance, not 100.